apps/roam/src/utils/internalError.ts

```diff
@@ -0,0 +1,69 @@
+import posthog from "posthog-js";
+import type { Properties } from "posthog-js";
+import renderToast from "roamjs-components/components/Toast";
+import { getVersionWithDate } from "~/utils/getVersion";
+import getCurrentUserDisplayName from "roamjs-components/queries/getCurrentUserDisplayName";
+import sendErrorEmail from "~/utils/sendErrorEmail";
+
+const internalError = ({
+  error,
+  userMessage,
+  type,
+  context,
+  sendEmail, // true by default
```

Comment:
This is only true for ln 48.

Use `sendEmail = true` to make the default explicit.  This aligns with [typescript patterns](https://www.typescriptlang.org/docs/handbook/2/functions.html#optional-parameters).